lib/HTMLOptionElement.js

```diff
@@ -22,7 +22,7 @@ module.exports = class HTMLOptionElement extends HTMLElement {
 
     if (value !== oldValue) {
       this._emitter.emit("_insert");
-      parent.dispatchEvent(new Event("change", { bubbles: true }));
+      if (parent) parent.dispatchEvent(new Event("change", { bubbles: true }));
```

Comment:
Tar gärna emot ett test på detta

Reply:
När jag tänker efter förstår jag nog inte use-caset här... Du kommer ju altid ha en parent eftersom detta ligger i `<body>`.

Reply:
Se testet. Skapar ett element utan att sätta in det. Då har det ingen parent, alltså ska inget event skickas.

Annat exempel är att i en browser göra:
```javascript
document.createElement("div").parentElement;

Reply:
Makes sense. Men jag förstår inte use-caset. Varför vill man ta ut referensen på `parentElement` eller skicka event till ett element som inte ligger i DOM:en?